aws-ecs/terraform/templates/controller_tls_task_def.tpl

```diff
@@ -0,0 +1,87 @@
+[
+  {
+    "name": "s3-downloader",
+    "image": "amazon/aws-cli:latest",
+    "cpu": 128,
+    "memory": 128,
+    "essential": false,
+    "command": ["s3","sync", "s3://${certificate_bucket_name}/${certificate_bucket_prefix}", "/var/controller-tls-secrets"],
```

Comment:
why s3 for certificates? why not ACM?

Reply:
It's build-in(container level) certificate for TLS server-mode, so there should be a file, we cannot export ACM certificates as file and it's optional.
Other option could be SSM parameter/secrets manager for example. I just thought that as we're using files in Docker/Helm examples so let it be here as well.
For ACM certificate used at load balancer level there's another  variable option used in datastore/controller_settings map.